include/sys/zfs_znode.h

```diff
@@ -258,6 +258,13 @@ zfs_inherit_projid(znode_t *dzp)
 #define	ZTOZSB(znode)	((zfsvfs_t *)(ZTOI(znode)->i_sb->s_fs_info))
 #define	ITOZSB(inode)	((zfsvfs_t *)((inode)->i_sb->s_fs_info))
 
+#define	ZTOTYPE(zp)	(ZTOI(zp)->i_mode)
+#define	ZTOGID(zp) (ZTOI(zp)->i_gid)
+#define	ZTOUID(zp) (ZTOI(zp)->i_uid)
+#define	ZTONLNK(type) (ZTOI(zp)->i_nlink)
```

Comment:
typo `ZTONLNK(type)` -> `ZTONLNK(zp)`.

Please add a new line before the `Z_IS*` block.